e2e/testcafe-devextreme/tests/editors/checkBox/common.ts

```diff
@@ -1,14 +1,26 @@
+/* eslint-disable no-restricted-syntax */
 import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
 import { Selector } from 'testcafe';
+import Guid from 'devextreme/core/guid';
 import url from '../../../helpers/getPageUrl';
 import { createWidget } from '../../../helpers/createWidget';
 import {
   appendElementTo,
-  insertStylesheetRulesToPage, setStyleAttribute,
+  insertStylesheetRulesToPage,
+  setClassAttribute,
+  setStyleAttribute,
 } from '../../../helpers/domUtils';
-import { testScreenshot } from '../../../helpers/themeUtils';
+import { getDarkThemeName, testScreenshot } from '../../../helpers/themeUtils';
+
+const valueModes = [false, true, undefined];
 
 const CHECKBOX_CLASS = 'dx-checkbox';
+const READONLY_STATE_CLASS = 'dx-state-readonly';
+const DEFAULT_STATE_CLASS = '';
+const ACTIVE_STATE_CLASS = 'dx-state-active';
+const HOVER_STATE_CLASS = 'dx-state-hover';
+const FOCUSED_STATE_CLASS = 'dx-state-focused';
+const DISABLED_STATE_CLASS = 'dx-state-disabled';
```

Comment:
There are no tests for checkboxes with scaling applied. Something like here: https://github.com/EugeniyKiyashko/DevExtreme/blob/88ae219ba389a4e89473e9c437f2d3ef97119c7f/e2e/testcafe-devextreme/tests/editors/radioGroup/common.ts#L91-L92